packages/riipen-ui/src/components/Switch.jsx

```diff
@@ -0,0 +1,213 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import _JSXStyle from "styled-jsx/style";
+
+import ThemeContext from "../styles/ThemeContext";
+
+const Switch = props => {
+  const {
+    checked,
+    classes,
+    color,
+    disabled,
+    id,
+    onChange,
+    required,
+    size,
+    value,
+    ...other
+  } = props;
+
+  const theme = React.useContext(ThemeContext);
+
+  return (
+    <span className={clsx(classes)}>
+      <input
+        checked={checked}
+        className={clsx(color, size)}
+        disabled={disabled}
+        id={id}
+        onChange={onChange}
+        required={required}
+        type="checkbox"
+        value={value}
+        {...other}
+      />
+      <style jsx>{`
+        input {
+          background-color: ${theme.palette.grey[400]};
+          cursor: pointer;
```

Comment:
I think we should also not have `cursor: pointer` if `other.readOnly` is `true` since it is unclickable in that case.

Reply:
Should we still have the shadow on hover if readOnly is true? @jordanell